src/python/WMCore/Credential/Proxy.py

```diff
@@ -159,6 +159,8 @@ def setEnv(self, cmd):
         if self.uisource is not None and len(self.uisource) > 0:
             ret = 'source ' + self.uisource + ' && ' + ret
         if self.cleanEnvironment:
+            # Need to escape ' in cmd
+            ret = ret.replace("'", "'\\''")
```

Comment:
The two instructions are equivalent, of course using `r'string'` is the standard way of telling python "do not escape the string", not escaping the escape character :)

Let's do the merge in this crab branch, and make sure the code is changed in the master pull.